vstools/functions/timecodes.py

```diff
@@ -22,11 +22,19 @@
 
 @dataclass
 class Timecode:
+    """A Timecode object representing the timecodes of a specific frame."""
+
     frame: int
+    """The frame number."""
+
     numerator: int
+    """The framerate numerator."""
```

Comment:
This is just the frame duration in milliseconds
which yeah you could argue it's what a framerate is composed of but frame length is easier to understand on a frame basis